### PR TITLE
adds event for Airbnb Escape Room

### PR DESCRIPTION
**Pull request by fboxwala**

### CODE REVIEW

File: content/W2016/W16-event-airbnb.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Title: Escape Room with Airbnb
+Date: 2016-02-15
+Category: Events
+Tags: social
+Slug: airbnb-escape-w16
+Author: Fatema Boxwala
+Summary: Hang out with some cool people from Airbnb while trying to escape
+some rooms!
```
Comment (ehashman) [inline content/W2016/W16-event-airbnb.md]: Same thing with summary here

File: content/W2016/W16-event-airbnb.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Have you been wondering what those “escape room” things everyone’s been talking
+about lately are? Come find out with WiCS and Airbnb!
+
+We’re going out to Adventure Rooms Canada to try it for ourselves, and maybe
+beat a record while we’re there.
+
+This event has limited attendance so [register soon](https://docs.google.com/forms/d/1v7OqMeQ0QI6RHETIIEA7VgoUeUm3VT7m0QbP0fPUtQM/viewform)!
+
+Trigger warning: this event involves some activities that might be disturbing
```
Comment (ehashman) [inline content/W2016/W16-event-airbnb.md]: I prefer "content warning" to trigger warning, as the former wording is more inclusive.

**Comment by fboxwala** [comment]
is this mergeable?

File: content/W2016/W16-event-airbnb.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+Title: Escape Room with Airbnb
+Date: 2016-02-15
+Category: Events
+Tags: social
```
Comment (ehashman) [inline content/W2016/W16-event-airbnb.md]: Needs an industry tag.

File: content/W2016/W16-event-airbnb.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+Title: Escape Room with Airbnb
+Date: 2016-02-15
```
Comment (ehashman) [inline content/W2016/W16-event-airbnb.md]: Seeing as all the PRs recently made aren't all going to go online at the same time, we may want to tweak publication times of these posts.

**Comment by ehashman** [comment]
Add that missing tag, update the date and I will be pleased :)

**Pull request merged**